uast/node.go

```diff
@@ -180,12 +197,23 @@ func (c *ObjectToNode) ToNode(v interface{}) (*Node, error) {
 		return nil, err
 	}
 
-	return c.toNode(root)
-}
+	nodes, err := c.toNodes(root)
```

Comment:
I don't see the point of changing from `toNode` to `toNodes`, to have a function that will always return a single node, just to check that there's only one node in the slice... I don't see how it simplifies the code, quite the opposite.

Reply:
it returns several nodes not only one.